composer.json

```diff
@@ -16,15 +16,17 @@
         "psr/log": "~1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.7|~5.0",
+        "phpunit/phpunit": "~4.8.27|~5.0",
```

Comment:
probbaly to make tests running with ``--prefer-lowest`` working when using the PHPUnit version installed through composer